solr/core/src/test/org/apache/solr/handler/component/FacetPivot2CollectionsTest.java

```diff
@@ -65,8 +65,12 @@ public class FacetPivot2CollectionsTest extends SolrCloudTestCase {
   public static void setupCluster() throws Exception {
     // create and configure cluster
     configureCluster(1)
-        .addConfig(COLL_A, configset("different-stopwords" + File.separator + COLL_A))
-        .addConfig(COLL_B, configset("different-stopwords" + File.separator + COLL_B))
+        .addConfig(
+            COLL_A,
+            configset("different-stopwords" + FileSystems.getDefault().getSeparator() + COLL_A))
```

Comment:
I think forward slash always works for Path.resolve, which is what will happen inside configset method with this string.  We'll verify this build on Windows before merging.